torch_xla/csrc/runtime/stablehlo_composite_helper.cc

```diff
@@ -208,28 +247,32 @@ class BuildStableHLOCompositePass : public mlir::OperationPass<mlir::ModuleOp> {
       return mlir::success();
     }
```

Comment:
I wonder should we assert and return `mlir::failure()` instead, the `metadata` should be from the `output_ops`, so they should describe the output.

Reply:
sg. will update it later.